### DISCUSSION TITLE
Any plans for the Windows 7 3D Aero flip switcher?

### DISCUSSION THREAD

**Discussion by XuanChen1234** [valinet/ExplorerPatcher#2572] [Ideas]
This is a great restoration project but the Windows 7 3D switcher would be SO much appreciated.

**Comment by pyrates999** [comment]
The code was completely removed from windows 10 according to this: https://answers.microsoft.com/en-us/windows/forum/all/can-i-get-the-aero-flip-3d-on-windows-10-it-was/a6dfea86-93b5-44c0-8c75-2791ff72cdc3

So probably not.
